src/microsoft_mcp/email_framework/css/email_compatibility.py

```
@@ -0,0 +1,305 @@
+"""
+Email client compatibility fixes and optimizations
+Ensures CSS works across all major email clients
+"""
+
+from typing import Dict, List, Tuple
```

Ruff (0.12.2)

`typing.Tuple` imported but unused

Remove unused import: `typing.Tuple`

(F401)